### ISSUE TITLE
Dropdown : Screen reader focus is not visible while navigating items after activating the 'Basic uncontrolled example' control. 

### ISSUE THREAD

**Issue by 999-Dev** [OfficeDev/office-ui-fabric-react#10922]
**User Experience:**
Screen reader users will be confused if focus will not be visible after activating the control and navigating by using up and down arrow keys.

**Test Environment:**
OS: Version: RS5_Prerelease(1903 OS Build 19005.1)
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown
Edge Browser Version:
Microsoft Edge: 44.19005.1.0
Microsoft EdgeHTML: 18.19005
Chrome Browser Version:
Chrome: 77.0.3865.120

**Usages:**
Basic Dropdowns > Multi-select uncontrolled example
Controlled multi-select Dropdown > Multi-select controlled example

**Repro Steps:**
1.https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown in Edge and turn on Narrator.
2.Navigate to 'Basic input' button and activate it.
3.Navigate to 'Dropdown' link and activate it.
4.'Dropdown' pane will be appeared.
5.Navigate to 'Multi-select uncontrolled example' combo box control and activate it.
6.Navigate by using up and down arrow keys and verify the issue.

**Actual Result:**
Screen reader focus does not visible on items. Screen reader announces the items name but focus is not visible.

**Expected Result:**
Screen reader focus and announcement should be in sync.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7Bb24201d1-2276-4471-8183-e1e5353031ad%7D

**Attachment:**
[OneDrive_2019-10-21.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3750897/OneDrive_2019-10-21.zip)

**Comment by cliffkoh** [comment]
This looks like a regression, checking
